builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -1068,7 +1088,18 @@ func buildPathGetIssuerCRL(b *backend, pattern string) *framework.Path {
 						Fields: map[string]*framework.FieldSchema{
 							"crl": {
 								Type:        framework.TypeString,
-								Description: ``,
+								Required:    false,
+							},
+							"http_content_type": {
+								Type:        framework.TypeString,
+								Required:    false,
+							},
+							"http_raw_body": {
+								Type:        framework.TypeString,
+								Required:    false,
+							},
+							"http_status_code": {
+								Type:        framework.TypeString,
```

Comment:
I believe these are dummy fields that will not show up in the final response, we should probably remove them from the schema.

Reply:
So, not having these ends up breaking some of the tests. b/c the responses can return either these fields or the other responses. Specifically it was breaking ones with the new callback. 

It does end up in the response that's getting returned to the test in this case.

Reply:
this one is a bit confusing, seems like the function can return non-json 
 https://github.com/hashicorp/vault/blob/29406c43a157d3d6541c4ed03a655ee28dcd08e8/builtin/logical/pki/path_fetch_issuers.go#L1174-L1178
so even though 1 function handles multiple paths, it can return different content types/response shapes

Reply:
Removed these.